doc/subsystems/shell.rst

```diff
@@ -25,27 +39,40 @@ modules can be available at the same time.
 
 ``MODULE_NAME COMMAND``
  One of the available modules is "KERNEL", for the Kernel module.  More
- information can be found in :c:macro:`SHELL_REGISTER`.
+ information about registering a new module and its commands can be
+ found in :c:macro:`SHELL_REGISTER`.
 
-Help commands
-=============
 
-``help``
- Prints the list of available modules.
+Kernel module commands
+----------------------
 
-``help MODULE_NAME``
- Prints the names of the available commands for the module.
+The Kernel module commands display useful kernel-specific information:
+
+``kernel version``
```

Comment:
better to show how to select the kernel module and then just issue the commands directly without the kernel prefix:

```
> select kernel
> threads

```